frameit/lib/frameit/config_parser.rb

```diff
@@ -87,6 +87,10 @@ def validate_values(values)
           if key == 'padding'
             UI.user_error! "padding must be type integer or pair of integers of format 'AxB'" unless value.kind_of?(Integer) || value.split('x').length == 2
           end
+
+          if key == 'show_complete_frame'
+            UI.user_error! "show_complete_frame must be true or false" unless !!value == value
```

Comment:
I just tried to replicate the existing code style and avoid objections from the hound. I have no attachment to that way of doing it.